docs/UserGuide.md

```diff
@@ -421,6 +421,7 @@ _Details coming soon ..._
 
 1. **When using multiple screens**, if you move the application to a secondary screen, and later switch to using only the primary screen, the GUI will open off-screen. The remedy is to delete the `preferences.json` file created by the application before running the application again.
 2. **If you minimize the Help Window** and then run the `help` command (or use the `Help` menu, or the keyboard shortcut `F1`) again, the original Help Window will remain minimized, and no new Help Window will appear. The remedy is to manually restore the minimized Help Window.
+3. **When sorting people with same name/deadline**: In rare cases where multiple people have the exact same name/deadline and you commit a sort operation on that field, modifying one of those clients' other fields (i.e their project status) might cause those with that same name/deadline to be shuffled. The list will still be correctly sorted, but it might be jarring to see. This bug will be addressed soon in an upcoming iteration.
```

Comment:
```suggestion
3. **When sorting people with same name/deadline**: In rare cases where multiple people have the exact same name/deadline and you perform a sort operation based on that field, modifying one of those clients' other fields (i.e their project status) might cause those with that same name/deadline to be shuffled. The list will still be correctly sorted among clients with different name/deadlines, but it might be jarring to see. This bug is an issue with javafx's implementation of `SortedList` and will be addressed soon in an upcoming iteration.
```